<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmTimeLine</class>
 <widget class="QWidget" name="frmTimeLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="TimeLine" name="timeLine1" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="TimeLine" name="timeLine2" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>110</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="TimeLine" name="timeLine3" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>210</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeLine</class>
   <extends>QWidget</extends>
   <header>timeline.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
